code/modules/clothing/spacesuits/void/event_vr.dm

```diff
@@ -212,21 +212,49 @@
 /obj/item/clothing/suit/space/void/excelsior
 	name = "\improper Excelsior voidsuit"
 	desc = "A space suit from a particular spaceship: Excelsior."
-	
+
 	icon = 'icons/inventory/suit/item_vr.dmi'
 	default_worn_icon = 'icons/inventory/suit/mob_vr.dmi'
 	icon_state = "excelsior"
-	
+
 	sprite_sheets = ALL_VR_SPRITE_SHEETS_SUIT_MOB
 	sprite_sheets_obj = ALL_VR_SPRITE_SHEETS_SUIT_ITEM
 
 /obj/item/clothing/head/helmet/space/void/excelsior
 	name = "\improper Excelsior voidsuit helmet"
 	desc = "A space helmet from a particular spaceship: Excelsior."
-	
+
 	icon = 'icons/inventory/head/item_vr.dmi'
 	default_worn_icon = 'icons/inventory/head/mob_vr.dmi'
 	icon_state = "excelsior"
-	
+
 	sprite_sheets = ALL_VR_SPRITE_SHEETS_HEAD_MOB
 	sprite_sheets_obj = ALL_VR_SPRITE_SHEETS_HEAD_ITEM
+
+
+/obj/item/clothing/suit/space/void/altevian_heartbreaker
+	name = "\improper heartbreaker voidsuit"
+	desc = "The altevians' newest iteration of their armored suits. This one is tailored for zero-g environments, and while it can function in an area with gravity, it'll put a strain on even the most athletic of individuals."
+
+	icon = 'icons/inventory/suit/item_vr_altevian.dmi'
+	default_worn_icon = 'icons/inventory/suit/mob_vr_altevian.dmi'
+	icon_state = "rig-heartbreaker"
+
+	armor = list("melee" = 90, "bullet" = 90, "laser" = 90, "energy" = 90, "bomb" = 90, "bio" = 100, "rad" = 80)
+
+	species_restricted = list(SPECIES_ALTEVIAN)
+	no_cycle = TRUE
+	slowdown = 2.5
```

Comment:
In the future if this becomes an actual, obtainable item, either the armor values should come down some or the slowdown should go up some. Beacher for reference has slowdown 6 and the same armor values.

As an adminspawn item it's fine though.

Reply:
Breacher is also unathi exclusive, who only get 1.5 slowdown from it due to inheirent trait. This suit is technically worse than breacher, both by not being a rig and by being exclusive to species that DOESNT reduce the slowdown.

Reply:
Fair enough. Didn't realize that unathi have enough natural slowdown reduction to make it only 1.5